api/src/domain-logic/en-vi.js

```diff
@@ -27,3 +27,25 @@ where word = ?`
     })),
   }
 }
+
+exports.count = async function count() {
+  const sql = 'SELECT COUNT(DISTINCT word) as count FROM words'
+
+  const summary = db.prepare(sql).get()
+
+  return summary.count
+}
+
+exports.getSuggestions = async function ({ word, limit = 10 }) {
+  let limitNumber = parseInt(limit)
```

Comment:
`parseInt(limit, 10)` always pass the radix parameter when using `parseInt`.
I thought ESLint should catch this 🤔  Also think about what if `limitNumber` is NaN. Maybe using `Number(limit)` is better.